src/tests/suite/daos_container.c

```diff
@@ -84,6 +84,8 @@ co_create(void **state)
 
 	/** destroy container */
 	if (arg->myrank == 0) {
+		/* XXX check if this is a real leak or out-of-sync close */
+		sleep(5);
```

Comment:
probably can remove this.
BTW, for original issue adding a sleep before cont_close maybe helpful, now not needed.

Reply:
Hm... the sleep() here may hide some race conditions. I would suggest to not add it.